download/tasks.py

```diff
@@ -167,11 +172,9 @@ def download_oa(ctx, oa_url, oa_filter, output_dir):
 
     # Flatten all .csv into output directory.
     print("Collect OpenAddresses data")
-
     ctx.run(f"mkdir -p {output_dir}")
-    ctx.run(f"rm -rf {output_dir}/*")
 
-    for filename in included_files:
-        flat_name = path.relpath(filename, oa_tmp_dir).replace("/", "__")
+    for piece in included_files:
+        flat_name = path.relpath(path.join(filename, piece), oa_tmp_dir).replace("/", "__")
```

Comment:
I don't understand why this `path.join` would be necessary:
```suggestion
        flat_name = path.relpath(piece, oa_tmp_dir).replace("/", "__")
```

Reply:
That's actually wrong (it would mostly affects log esthetics though), I wanted to eliminate possible name collisions between two different OA source. I actually meant to do something like this: 

```suggestion
        flat_name = path.join(filename, path.relpath(piece, oa_tmp_dir)).replace("/", "__")
```